wb_animation_action/utils/git.py

```diff
@@ -28,7 +28,7 @@ def _init():
     result = subprocess.run('git config --list | grep user.name', shell=True, check=False)
     if result.returncode != 0:
         email = '{}+{}@users.noreply.github.com'.format(user_info['id'], username)
-        subprocess.check_output(['git', 'config', '--global', 'user.name', user_info['name']])
+        subprocess.check_output(['git', 'config', '--global', 'user.name', user_info['name'] or username])
```

Comment:
```suggestion
        name = user_info['name'] if ('name' in user_info and user_info['name']) else username
        subprocess.check_output(['git', 'config', '--global', 'user.name', name)
```
I afraid the `name` key might not exist either, what about this?

Reply:
If I am correctly interpreting [GH's user API docs](https://docs.github.com/en/rest/reference/users#get-a-user), the `user` field is guaranteed to exist. I have checked this code with the `pal-admin` user account, which lacks a real username at time of writing:

```py
import requests
import subprocess

user_info = user_info = requests.get('https://api.github.com/users/pal-admin').json()
print(user_info['name'] is None) # True
subprocess.check_output(['git', 'config', '--global', 'user.name', user_info['name']]) # TypeError
```

Reply:
I see. It is fine as is then